kasa/tests/newfakes.py

```diff
@@ -302,7 +306,9 @@ async def query(self, request, retry_count: int = 3):
 class FakeSmartTransport(BaseTransport):
     def __init__(self, info):
         super().__init__(
-            "127.0.0.123",
+            cparams=ConnectionParameters(
+                "127.0.0.123", credentials=Credentials("", "")
```

Comment:
I'm seeing quite a bit of `Credentials("", "")` everywhere which doesn't look nice. Maybe `Credentials()` should just create the object using those as default values?

Reply:
Ok will do